include/slb/type_traits.hpp

```diff
@@ -403,7 +403,15 @@ namespace slb {
 
 // 23.15.3, helper class
 
-#if __cpp_lib_integral_constant_callable // C++14
+#if defined(__GLIBCXX__)
+#define SLB_HAS_STD_INTEGRAL_CONSTANT __has_include(<filesystem>) // libstdc++-8
+#elif __cpp_lib_integral_constant_callable // C++14
```

Comment:
The use of `HAS` here seems inappropriate, since we always require implementations _having_ `integral_constant`.

The include check is independent of standard modes. In C++11 mode, libstdc++8 `integral_constant` won't be callable.

Reply:
oh, that was fast. I already reformulated this hehe